calc.py

```diff
@@ -0,0 +1,82 @@
+# ! python 3
+# This program will take inputs one at a time from the user and do the operation
+# instructed in the order it was given then print out that function with the result.
+
+# Function that asks for input and tests if it's a number or not
+def getNums(message):
+    while True:
+        try:
+            userInput = int(input(message))
+        except ValueError:
+            print("Sorry, that wasn't a number, try again")
+            continue
+        else:
+            return userInput
+            break
+# Get how many numbers are we going to do operations to
+qty = getNums('How many numbers are we dealing with ')
+
+# create blank lists, one for numbers and the other for operators
+numLst = []
+opLst = []
+# Use natural language for the input. Discerns between the first, last or other numbers
+for i in range(int(qty)):
+    if i == 0:
+        numLst.append(getNums("First Number "))
+    elif i == int(qty) - 1:
+        numLst.append(getNums("Last Number "))
+    else:
+        numLst.append(getNums("Next number "))
+# Only ask for what operation we want done if it's not the last number
+    if i != (int(qty)) - 1:
+        while True:
```

Comment:
you don't need this other `while True` conditional because you're already in a conditional from line 31. Instead what you could say is
`while i != (int(qty)) - 1:`
http://www.tutorialspoint.com/python/python_while_loop.htm